File: app/src/main/java/com/vano/myrestaurant/controller/fragment/OrdersFragment.java
package com.vano.myrestaurant.controller.fragment;

import android.os.Bundle;

import androidx.fragment.app.ListFragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ArrayAdapter;

import com.vano.myrestaurant.model.entity.Order;
import com.vano.myrestaurant.model.service.OrderService;

public class OrdersFragment extends ListFragment {

    private OrderService orderService;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        orderService = new OrderService(getContext());

        final ArrayAdapter<Order> adapter = new ArrayAdapter<>(getContext()
                , android.R.layout.simple_list_item_1
                , orderService.readAllOrders());

        setListAdapter(adapter);
        return super.onCreateView(inflater, container, savedInstanceState);
    }
}

File: app/src/main/java/com/vano/myrestaurant/controller/fragment/OrdersFragment.kt
package com.vano.myrestaurant.controller.fragment

import com.vano.myrestaurant.model.service.OrderService
import android.view.LayoutInflater
import android.view.ViewGroup
import android.os.Bundle
import android.view.View
import android.widget.ArrayAdapter
import androidx.fragment.app.ListFragment

class OrdersFragment : ListFragment() {

    private var orderService: OrderService? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        orderService = OrderService(context)
    }

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        val adapter = context?.let {
            ArrayAdapter(
                it, android.R.layout.simple_list_item_1, orderService?.readAllOrders().orEmpty()
            )
        }

        listAdapter = adapter

        return super.onCreateView(inflater, container, savedInstanceState)
    }
}